src/castra/core.cljs

```diff
@@ -80,12 +80,15 @@
                          (when x (.setItem js/localStorage storage-key x))))
 
 (defn- ajax
-  [{:keys [ajax-fn clj->json json->clj on-error] :as opts} expr]
+  [{:keys [ajax-fn clj->json json->clj on-error additional-headers] :as opts} expr]
   (let [prom    (.Deferred js/jQuery)
         headers (-> {"X-Castra-Csrf"          "true"
                      "X-Castra-Tunnel"        "transit"
                      "X-Castra-Validate-Only" (str (boolean *validate-only*))
                      "Accept"                 "application/transit+json"}
+                    (merge (if (instance? javeline.core.Cell additional-headers)
```

Comment:
```suggestion
                    (merge (if (instance? javelin.core.Cell additional-headers)
```

@rwillig I found a typo :)

Reply:
holy crap, how did that get past me?

Reply:
that's actually weird bc, it must  be broken in our submodule, yet it works?

Reply:
oh nvm. i had just copied the file over from our submodule but i probably didn't have our latest branch checked out